src/Orleans/Serialization/SerializationContext.cs

```diff
@@ -52,6 +55,16 @@ public interface ISerializationContext : ISerializerContext
         int CheckObjectWhileSerializing(object raw);
 
         int CurrentOffset { get; }
+
+        void SerializeInner(object obj, Type expected);
+    }
+
+    public static class ISerializationContextExtensions
```

Comment:
I get why, but maybe this should be just `SerializationContextExtensions` to avoid IDE squiggles.